tests/console/zypper_lr.pm

```diff
@@ -16,6 +16,8 @@ use testapi;
 use utils;
 
 sub run() {
+    # ZYPPER_LR is needed for inconsistent migration, test would fail looking for deactivated addon
+    set_var 'ZYPPER_LR';
```

Comment:
strange variable , it is exsists only in single place , probably cleaner approach would be drop it and control same thing with MIGRATION_INCONSISTENCY_DEACTIVATE ?